src/main.cpp

```diff
@@ -4769,22 +4862,22 @@ bool static ProcessMessage(CNode* pfrom, std::string strCommand, CDataStream& vR
 
         CBlockIndex *pindexLast = NULL;
         BOOST_FOREACH(const CBlockHeader& header, headers) {
-            CValidationState state;
-            if (pindexLast != NULL && header.hashPrevBlock != pindexLast->GetBlockHash()) {
-                Misbehaving(pfrom->GetId(), 20);
-                logWarning(Log::Net) << "p2p HEADERS command received non-continues headers sequence from" << pfrom->GetId();
-                return false;
-            }
-            if (!AcceptBlockHeader(header, state, chainparams, &pindexLast)) {
-                int nDoS;
-                if (state.IsInvalid(nDoS)) {
-                    if (nDoS > 0)
-                        Misbehaving(pfrom->GetId(), nDoS);
-                    logWarning(Log::Net) << "p2p HEADERS command received invalid header from" << pfrom->GetId();
-                    return false;
-                }
-            }
-        }
+                        CValidationState state;
```

Comment:
ditto. Too much whitespace changed.

Reply:
hmm, thought the project was using spaces, I see the line above has tabs.

How do you have your IDE configured to handle indentation? tabs? tabs turn into <n?> whitespaces?

Reply:
Ah, tabs, that explains that..

My IDE (QtCreator) has the default-setting it only fixes whitespace on lines I changed anyway.

Fixing tabs should should interpret those tabs as 4 spaces.

Naturally, when the final code moves to another file, you'll end up touching lots of lines. But since your auto-indent for boost_foreach doesn't work, please make it look good manually :)